### PR TITLE
pragma: add minimalist message pragma implementation

### PR DESCRIPTION
**Pull request by pmjdebruijn**
for example:

#pragma message ("in a bottle")

results in:

hello.c(207): Note: in a bottle

### CODE REVIEW

**Comment by pmjdebruijn** [comment]
Essentially #460 reworked.

File: src/cc65/pragma.c [hunk 1]
```diff
@@ -750,6 +752,13 @@ static void IntPragma (StrBuf* B, IntStack* Stack, long Low, long High)
 
 
 
+void MakeMessage (const char* Message)
```
Comment (greg-king5) [inline src/cc65/pragma.c]: It should be defined as a `static` function.

**Comment by pmjdebruijn** [comment]
Amended.

**Comment by oliverschmidt** [comment]
There needs to be documentation in the section http://cc65.github.io/doc/cc65.html#s7

Hint: The source is at https://github.com/cc65/cc65/blob/master/doc/cc65.sgml#L956

**Comment by pmjdebruijn** [comment]
Of course, added :)

File: doc/cc65.sgml [hunk 2]
```diff
@@ -1119,6 +1119,21 @@ parameter with the <tt/#pragma/.
...
+<sect1><tt>#pragma message (&lt;message&gt;)</tt><label id="pragma-message"><p>
+
+  This pragma is used to display informational messages at compile-time.
+
+  Example:
+  <tscreen><verb>
+        #pragma message ("foo")
+  </verb></tscreen>
+
+  Results in the compiler outputting the following on the console:
```
Comment (oliverschmidt) [inline doc/cc65.sgml]: I presume that Greg comes up with a proposal for a different wording...

Comment (pmjdebruijn) [inline doc/cc65.sgml]: I'd be happy to amend it whatever way you see fit.

Comment (greg-king5) [inline doc/cc65.sgml]: Change "`on the console`" above into "`to stderr`".

I like your allusion, in the commit message, to _Message in a Bottle_.  I would like to see it in this document, too.

Above that example, you should mention that the message must be a string literal.

Comment (pmjdebruijn) [inline doc/cc65.sgml]: Amended.

Comment (pmjdebruijn) [inline doc/cc65.sgml]: Did anybody notice the completely random selection of a line number example of 42 :)

**Comment by oliverschmidt** [comment]
This seems ready to be merged...

File: doc/cc65.sgml (hunk 2 above)
Comment (greg-king5) [inline doc/cc65.sgml]: > Did anybody notice the completely random selection of a line number example of 42? :-)

:computer: :grinning:
I confess that I didn't recognize that one.

**Pull request merged**